src/test/java/com/epam/tc/hw1/AddOperations.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import com.epam.tc.hw1.dataprovider.DataOperations;
+import org.testng.annotations.Test;
+
+public class AddOperations {
+
+    private final Calculator calculator = new Calculator();
```

Comment:
I suppose that this field could be moved to the base test class.
Also it will be better create instance of calculator before each test in `@Before` hook and close it in `@After` hook

Reply:
I created class "BaseTest" where I declared instance of calculator, @BeforeTest and @AfterTest methods.